src/update/UpdateManager.cpp

```diff
@@ -146,7 +146,10 @@ void UpdateManager::GetUpdates(bool ignoreNetworkErrors)
                             info.GetOperatingSystemId() & wxOS_MAC)
                         {
                             if (wxFileName(mAudacityInstallerPath).Exists())
-                                wxTheApp->CallAfter([this] { wxExecute(mAudacityInstallerPath, wxEXEC_ASYNC); });
+                            {
+                                std::string cmd = info.GetOperatingSystemId() & wxOS_MAC ? "Open " + mAudacityInstallerPath : mAudacityInstallerPath;
+                                wxTheApp->CallAfter([cmd, this] { wxExecute(cmd, wxEXEC_ASYNC); });
+                            }
```

Comment:
It looks like this commit corrects the previous commit, which was not actualy correct for macOS, right?

Squash the two commits. Make a sequence of commits that are stages of implementation that are each buildable and runnable without errors.  But don't leave a sequence of known bugs and corrections.